level1a/handlers/level1a.py

```diff
@@ -252,12 +270,22 @@ def lambda_handler(event: Event, context: Context):
             raise DoesNotCover("Attitude data is missing timestamps")
         if not covers(orbit_df.index, min_time, max_time):
             raise DoesNotCover("Orbit data is missing timestamps")
-        if not covers(htr_df.index, min_time, max_time):
-            raise DoesNotCover("HTR data is missing timestamps")
 
-        attitude_subset = interpolate(attitude_df, rac_df.index)
-        orbit_subset = interpolate(orbit_df, rac_df.index)
-        htr_subset = interpolate(htr_df, rac_df.index)
+        attitude_subset = interpolate(
+            attitude_df,
+            rac_df.index,
+            max_diff=get_offset(ATTITUDE_FREQUENCY),
+        )
+        orbit_subset = interpolate(
+            orbit_df,
+            rac_df.index,
+            max_diff=get_offset(ORBIT_FREQUENCY),
+        )
+        htr_subset = interpolate(
+            htr_df,
+            rac_df.index,
+            max_diff=get_offset(HTR_FREQUENCY),
+        )
```

Comment:
I'm wondering if we want the same functionality for all of these... Is platform data not a strict requirement for L1A?

Reply:
It is. Best would be to drop any rows with `nan`, or at least rows where the platform-data is/contains `nan`. Maybe there is a simple filtering that could be performed before saving? Something like this:

```
out_data = out_data[all(isfinite(outdata.PlatformParameter))]
```

maybe?

Or maybe this is better left until #6 ?